webapp-client/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Lobster Ice Cream</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
is everything in the body except for <div id="root"></div> deleteable?

Reply:
A lot of stuff here controls what the tab of this website looks like, but we are going to clean her up a bit.